adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -525,11 +517,20 @@ def adjust_touch_threshold(self, adjustment: int) -> None:
               if cp.touch_A1:
                   print('Touched pad A1')
         """
-        for touch_in in self._touches:
-            if isinstance(touch_in, touchio.TouchIn):
-                touch_in.threshold += adjustment
+        for touch_in in self._touches.values():
+            touch_in.threshold += adjustment
         self._touch_threshold_adjustment += adjustment
 
+    @property
+    def touch_pins(self) -> List[Pin]:
+        """Return a list of all the pins that are set up as touchpad inputs"""
```

Comment:
Property doc style is not to use "Returns"
```suggestion
        """A list of all the pins that are set up as touchpad inputs"""
```